src/site/content/en/connect/index.njk

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: Connect
+description: |
+  TODO
+pagination:
```

Comment:
Instead of using Eleventy's pagination here, I think we may want to loop over the authors collection itself. I think Eleventy's pagination will try to generate new pages for us but we probably want to have a single page with all of the connect folks on it so we can easily filter the page.

So you'd remove the pagination yaml and do:

```njk
<div class="w-grid__columns w-grid__columns--three" role="list">
{% for author in collections.authors %}

{% if author.connect %}

{% endif %}
{% ConnectCard author %}
{% endfor %}
</div>
```